core/tests/protractor_desktop/stateEditor.js

```diff
@@ -522,6 +523,55 @@ describe('State editor', function() {
     users.logout();
   });
 
+  it('should generate warning message if card height limit is exceeded ',
+    function() {
+      users.createUser('user@heightWarning.com', 'userHeightWarning');
+      users.login('user@heightWarning.com');
+
+      workflow.createExploration();
+
+      var postTutorialPopover = element(by.css('.popover-content'));
```

Comment:
I don't think you'll have to handle all these cases here (see editorFeatures.js around line 923).

Reply:
Hi @DubeySandeep, 

After creating an exploration, we try to set content using the function defined [here](https://github.com/oppia/oppia/blob/develop/core/tests/protractor_utils/ExplorationEditorMainTab.js#L460). In this function all these cases will be handled automatically.

In the test defined for height watching check, I do not want to set the content since if the content is saved, the warning will not be visible. So, all these cases were handled separately inside the test.

I am not sure if I have correctly understood your query here. Do let me know your inputs here.

Thanks!

Reply:
Ah, I see thanks for the explanation.